stages/wrapper_ci.yml

```diff
@@ -33,24 +44,38 @@ stages:
   - template: ./wrapper_stage.yml
     parameters:
       jobs:
-        - template: ../jobs/wrapper_job.yml
-          parameters:
-            container: ${{ parameters.container }}
-            name: ${{ parameters.validateName }}
-            steps:
-              - checkout: self
-                clean: true
-                persistCredentials: true
-              - template: ../steps/check_branch_name.yml
-              - ${{ if parameters.azureAuth }}:
-                - template: ../steps/azure_auth.yml
+        - ${{ if ge(length(parameters.validationJobs), 1) }}:
+          - ${{ each job in parameters.validationJobs }}:
+            - template: ../jobs/wrapper_job.yml
+              parameters:
+                condition: ${{ job.condition }}
+                container: ${{ job.container }}
+                name: ${{ job.name }}
+                steps: ${{ job.steps }}
```

Comment:
Any default steps we could add here? E.g. Checkout, Login, etc. (similar to how the single job scenario works)

Reply:
Done